tests/data_replicator.rs

```diff
@@ -71,7 +71,7 @@ where
 }
 /// ring a -> b -> c -> d -> e -> a
 #[test]
-fn gossip_ring() {
+fn gossip_ring() -> Result<(), solana::result::Error> {
```

Comment:
where do we define solana::result::Error?  just curious, might be useful for #822

Reply:
You can tighten up that code with:

```rust
use solana::result;
fn gossip_ring() -> result::Result {
```